packages/pn-pa-webapp/src/utils/AppError/PAAppErrorFactory.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  AppError,
+  AppErrorFactory,
+  ServerResponseError,
+  UnknownAppError,
+} from '@pagopa-pn/pn-commons';
+
+import { GenericInvalidParameterAppError } from './GenericInvalidParameterAppError';
+import { ServerResponseErrorCode } from './types';
+
+export class PAAppErrorFactory extends AppErrorFactory {
+  private translateFunction: (path: string, ns: string) => string = (path: string) => path;
```

Comment:
why the default translate function? If it's for test purposes, please add a comment about it. Otherwise, I think I would prefer the app to break if no translation function has been provided to the error factory.

Reply:
It's not for test purpose, but if you check the constructor you see that we are forced to pass a translate function. So the app cannot break.
On the other side the valorization inside the class is useless and maybe the definition it's enough